.pre-commit-config.yaml

```diff
@@ -7,9 +7,9 @@ repos:
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
 -   repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.12.2
+    rev: v0.11.2
```

Comment:
this is the version in the dev guide https://github.com/scikit-learn/scikit-learn/blob/a9876111036fca025c26448eff072b3ab6e6208b/doc/developers/development_setup.rst#L132